feat: add anime search

Plyrs1 opened the pull request.

Plyrs1 committed:
feat: add anime search

Plyrs1 committed:
fix: use discord.js color constants

Plyrs1 committed:
fix: only show manga in anime command

Plyrs1 committed:
refactor: move anime embed builder to separate file

- add emojis to links

Plyrs1 committed:
fix: get available color from constants

- fix color type cast

satoufuyuki commented on src/utils/Anilist.ts:
```diff
@@ -0,0 +1,171 @@
+
+export class Anilist {
+    public constructor(private readonly token?: string) {
+        this.token = token;
```
```suggestion
```
unnecessary

satoufuyuki commented on src/utils/Anilist.ts:
```diff
@@ -0,0 +1,171 @@
+        return new Promise((resolve, reject) => {
+            // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+            if (this.token) options.headers["Authorization"] = `Bearer ${this.token}`;
+            const req = request(options, res => {
```
use got

Plyrs1 committed:
fix: use got for api request

- replace try catch with sapphire/result
- truncate text to conform character limits
